dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoTagResource.java

```diff
@@ -244,6 +245,7 @@ public Response deleteTags(@ApiParam(hidden = true) @Auth User user,
     @POST
     @Timed
     @UnitOfWork
+    @Beta
```

Comment:
I did not know there was a Beta annotation. I wonder if we should add this to the permissions APIs, since it's possible we may discover some shortcomings? Does this get reflected in Swagger somehow?

Reply:
I don't think it gets reflected in Swagger, it does get picked up in IntelliJ.